floris/core/turbine/tum_operation_model.py

```diff
@@ -0,0 +1,986 @@
+from __future__ import annotations
+
+import copy
+import os
+from pathlib import Path
+
+import numpy as np
+from attrs import define
+from scipy.interpolate import RegularGridInterpolator
+from scipy.optimize import fsolve
+
+from floris.core.rotor_velocity import (
+    average_velocity,
+    compute_tilt_angles_for_floating_turbines,
+    rotor_velocity_air_density_correction,
+)
+from floris.core.turbine.operation_models import BaseOperationModel
+from floris.type_dec import (
+    NDArrayFloat,
+    NDArrayObject,
+)
+
+
+@define
+class TUMLossTurbine(BaseOperationModel):
+    """
+    Static class defining a wind turbine model that may be misaligned with the flow.
+    Nonzero tilt and yaw angles are handled via the model presented in
+    https://doi.org/10.5194/wes-2023-133 .
+
+    The method requires C_P, C_T look-up tables as functions of tip speed ratio and blade pitch
+    angle, available here:
+    "../floris/turbine_library/LUT_iea15MW.npz" for the IEA 3.4 MW (Bortolotti et al., 2019)
+    As with all turbine submodules, implements only static power() and thrust_coefficient() methods,
+    which are called by power() and thrust_coefficient() on turbine.py, respectively.
+    There are also two new functions, i.e. compute_local_vertical_shear() and control_trajectory().
+    These are called by thrust_coefficient() and power() to compute the vertical shear and predict
+    the turbine status in terms of tip speed ratio and pitch angle.
+    This class is not intended to be instantiated; it simply defines a library of static methods.
+
+    TODO: Should the turbine submodels each implement axial_induction()?
+    """
+
+    @staticmethod
```

Comment:
Are these actually used as static methods?

Reply:
The class is never actually instantiated anywhere. However, these could just as easily be standalone modules on the function (or, they could be left on the class, but the `@staticmethod` could be removed---that works, but my linter doesn't like that the first argument isn't `self`). 

Do you have an opinion on that? Should they just be standalone functions?

Reply:
It seems that the `@staticmethod` decorator is appropriate here, but I've now made all functions on this module static methods of `TUMLossModel`